crates/re_types/src/components/view_coordinates_ext.rs

```diff
@@ -6,7 +6,16 @@ use crate::view_coordinates::{Axis3, Handedness, Sign, SignedAxis3, ViewDir};
 
 use super::ViewCoordinates;
 
+impl Default for ViewCoordinates {
+    fn default() -> Self {
+        Self::DEFAULT
+    }
+}
+
 impl ViewCoordinates {
+    /// The `ViewCoordinates` used when none have been specified.
+    pub const DEFAULT: Self = Self::RUB;
```

Comment:
This will be confusing. Default for what? Cameras? Images? 2D spaces? 3D spaces? Cars? Who's default? Rerun should not be opinionated about these things.

For instance: I would argue it makes sense to have +Z be the default up-axis in the world coordinate system, regardless of what your camera coordinates are.

For reference: arktiscenes and open_phtotogrammetry_format both use Z+ as world-up, while objectron uses Y+. So thats two against one. I like those odds!